### ISSUE TITLE
Helper emitted before goog.module

### ISSUE THREAD

**Issue by ConsciousCode** [angular/tsickle#869]
As the title says, when using ES6 features (in my case, async) which generate helpers, the helpers are emitted before goog.module. The closure compiler complains that goog.module isn't the first statement. I'll probably be able to get around this by fiddling with the output language so it just uses async natively, but someone who wants to emit ES5 or lower will have to add a transpilation step from ES6 emitted from the closure compiler to their target. Is there a good way to guarantee that goog.module comes before helpers so this isn't necessary?

**Comment by evmar** [comment]
How are you invoking tsickle?  We recently changed things so that we expect you to build with the `--importHelpers` flag.

**Comment by ConsciousCode** [comment]
Ah ok, I wasn't using the flag. I guess it forces the helpers to be included like normal modules and not emitted at the beginning? I think this solves my problem, though it might be a good idea to add a warning if it's not included, or maybe make it enabled by default.

**Comment by evmar** [comment]
Yeah I agree.  That's why I was asking how you invoke tsickle -- trying to reason through which layer we should be checking this at.
